oddt/surface.py

```diff
@@ -0,0 +1,136 @@
+"""This module generates, plots, and does computation with molecular surfaces.
+"""
+
+import oddt
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d.art3d import Poly3DCollection
+from skimage import measure
+from skimage.morphology import ball, binary_closing
+from scipy.spatial.distance import cdist
+
+
+def generate_surface_marching_cubes(molecule, remove_hoh=False, scaling=1, probe_radius=1.4):
+    """Generates a molecular surface mesh using the marching_cubes_lewiner method from scikit-image.
```

Comment:
I think those lines could be a bit shorter. I don't see why we shouldn't adhere to PEP8